app/controllers/import_users_controller.rb

```diff
@@ -56,6 +56,13 @@ def valid_input_file?(imported_users, file) # rubocop:disable all
       redirect_to(:back) && return
     end
 
+    # make sure that there's spaces in the header line
+    if imported_users.first.keys.join('').match(/\s/)
```

Comment:
Just to double-check, this looks for spaces *anywhere* inside the headers (not just at the ends). That seems good to me, I just wanted to make sure that I understood what was going on.